### DISCUSSION TITLE
Pre-Release v2.3.8883

### DISCUSSION THREAD

**Discussion by gbakeman** [nutdotnet/WinNUT-Client#148] [Announcements]
## What's Changed
* Fix 'Create LogFile' option is on, but the option is not working when program is running. by @MaxMadcc in https://github.com/nutdotnet/WinNUT-Client/pull/147

**Full Changelog**: https://github.com/nutdotnet/WinNUT-Client/compare/v2.3.8834...v2.3.8883

<hr /><em>This discussion was created from the release <a href='https://github.com/nutdotnet/WinNUT-Client/releases/tag/v2.3.8883'>Pre-Release v2.3.8883</a>.</em>

**Comment by masterwishx** [comment]
@gbakeman I'm very sorry had no chance to check the pr, but pre-release seems working fine for this fix...

**Reply by gbakeman** [reply]
No worries at all, thank you for the feedback!
